ui/lib/ldap/addon/components/page/overview.hbs

```diff
@@ -57,6 +57,7 @@
             @selectLimit="1"
             @fallbackComponent="input-search"
             @onChange={{this.selectRole}}
+            @renderInPlace={{true}}
```

Comment:
<img width="498" alt="Screenshot 2024-05-03 at 10 35 38" src="https://github.com/hashicorp/vault/assets/82459713/a69c55cb-334a-4bc1-8630-c497d0c831ca">
(without the div around the button, the button height was off)